Marketplace/Providers/getting-started-guide-marketpalce-ecosystem-provider-portal.md

```diff
@@ -0,0 +1,236 @@
+{{{
+"title": "Getting Started Guide: Marketplace & Ecosystem Provider Portal",
+"date": "02-02-2017",
+"author": "",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+The Ecosystem Provider Portal &ndash; or EPP &ndash; is an application that allows CenturyLink Cloud Provider Partners to manage the display of their products on the CenturyLink Marketplace. Once an Ecosystem Provider has certified a product, a member of the Ecosystem Team will provide the partner with a welcome email with the credentials to log-in to EPP.
+
+### Signing In
+
+Sign into the [Provider Portal](http://provider-portal.staging.ctl.io/#/login) using your credentials. You will be redirected to the Welcome screen.
+
+![EPP1](../images/EPP1.png)
```

Comment:
Change this link to `../../images/EPP1.png`
Change other image links accordingly.

Since this file is located in the `Marketplace/Providers` directory, you need to go up two levels to get to the `images` directory.